src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { Person, PersonRole } from '@hmcts/rpx-xui-common-lib';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+
+import { ExclusionNavigationEvent, ExclusionState, ExclusionStateData } from '../../../models';
+import { ExclusionNavigation } from '../../../models/exclusion-navigation.interface';
+import * as fromFeature from '../../../store';
+
+@Component({
+  selector: 'exui-specific-access-approved',
+  templateUrl: './specific-access-approved.component.html'
+})
+export class SpecificAccessApprovedComponent implements OnInit {
+  @Input() public navEvent: ExclusionNavigation;
+  public domain = PersonRole.ALL;
+  public formGroup: FormGroup = new FormGroup({});
+  public personName: string;
+  public person: Person;
+  public subscription: Subscription;
+  public personRole: PersonRole;
+  public services: string[];
+  public assignedUser: string;
+
+  constructor(private readonly store: Store<fromFeature.State>) {
+  }
+
+  public ngOnInit(): void {
+    this.subscription = this.store.pipe(select(fromFeature.getRoleAccessState)).subscribe(exclusion => this.setPerson(exclusion));
+  }
+
+  private setPerson(exclusion: ExclusionStateData): void {
```

Comment:
is method `setPerson` needed?

Reply:
Please re-review - I've cleaned the unnecessary code from the file that I missed.